scripts/linters/pylint_extensions.py

```diff
@@ -1673,6 +1673,16 @@ class SingleLineCommentChecker(checkers.BaseChecker):
             'Please use a capital letter at the beginning of comment.',
             'no-capital-letter-at-beginning',
             'Please use capital letter to begin the content of comment.'
+        ),
+        'C0019': (
```

Comment:
Note that msg ids are used across all pylint extensions, they are not specific to a class here, they have to unique globally in this file. Please use the msg id displayed on the top of the file `C0039` onwards, and update the same on top indicating the msg id to use next.